### ISSUE TITLE
Fix teleport-ent-updater publishing for yum/zypper

### ISSUE THREAD

**Issue by bernardjkim** [gravitational/teleport#41789]
### Expected behavior:
Installing the teleport-ent-updater package via yum should install a yum package.

### Current behavior:
In some cases, yum may install a zypper package. This seems to be an issue resulting from the teleport-ent-updater package publishing workflow. Both zypper and yum packages are published to both zypper and yum repos.

### Bug details:
- Teleport version: 15.3.5
- Recreation steps: Observed on Amazon Linux 2 after following installation method via package manager https://goteleport.com/docs/installation/.
- Debug logs
```
bash-4.2# yum install teleport-ent-updater-15.3.5
Loaded plugins: ovl, priorities
Resolving Dependencies
--> Running transaction check
---> Package teleport-ent-updater.x86_64 0:15.3.4-1 will be updated
---> Package teleport-ent-updater.x86_64 0:15.3.5-1 will be an update
--> Finished Dependency Resolution

Dependencies Resolved

=======================================================================================================================================================================
 Package                                          Arch                               Version                                Repository                            Size
=======================================================================================================================================================================
Updating:
 teleport-ent-updater                             x86_64                             15.3.5-1                               teleport                              15 k

Transaction Summary
=======================================================================================================================================================================
Upgrade  1 Package

Total download size: 15 k
Is this ok [y/d/N]: y
Downloading packages:
Delta RPMs disabled because /usr/bin/applydeltarpm not installed.
teleport-ent-updater-15.3.5.amd64.zypper.rpm
...

bash-4.2# cat /etc/teleport-upgrade.d/installer
zypper
```

**Comment by thameezb** [comment]
Facing this issue

**Comment by bernardjkim** [comment]
Patch fix should be available in the next release v16.1.1.
